### DISCUSSION TITLE
I updated the latest 0.16 No emission sound

### DISCUSSION THREAD

**Discussion by ydms** [egzumer/uv-k5-firmware-custom#83] [General]
I updated the latest 0.16

But I found out that my K5

No emission sound

No sound when setting ROGER or MDC1200

**Comment by ErikS-web** [comment]
What firmware are u using. A (daily build) version from Telegram/Internet or

the Official release here https://github.com/egzumer/uv-k5-firmware-custom/releases/tag/v0.16

Did you change parameters with Menu 41.60 ?

Just tested here and works fine.... At TX audio works fine and at the end of TX an Two-tone as Roger or a burst for MDC1200

**Comment by egzumer** [comment]
You won't hear the sound in the speaker. The sound is sent through RF path only.
